charts/enterprise-kyverno-operator/README.md

```diff
@@ -2,6 +2,9 @@
 Nirmata Kyverno Opertor is a Kubernetes Operator to manage lifecycle of Kyverno, Adapters and Nirmata supported policies. 
 
 ## Prerequisites
+
+As a prerequisite, user needs to install [kyverno-operator-crd](../enterprise-kyverno-operator-crd/) to install operator.
```

Comment:
this is not a prerequisite
this is if the user wants to ensure delete of crds is done

Reply:
But we are moving the CRD inside the different chart so if a user go ahead by directly installing the operator chart, he/she will caught up in a error

Reply:
we didnt remove anything from the operator chart right?

if the user doesnt do this they will still have the old flow

Reply:
we removed the CRD from operator chart and moved it into the separate chart

Reply:
got it.